torch/onnx/symbolic_opset10.py

```diff
@@ -0,0 +1,40 @@
+import torch
+import torch.onnx
+# This import monkey-patches graph manipulation methods on Graph, used for the
+# ONNX symbolics
+import torch.onnx.utils
+
+from torch.onnx.symbolic_helper import parse_args, _unimplemented, _black_list_in_opset
+import torch.onnx.symbolic_opset9
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 10
+# Opset 10 is supported by ONNX release 1.5.0
+# release on 04/24/19
+
+
+# Blacklist operators for this opset version.
+# These operators have been updated in ONNX but not re-implemented here.
+# It is very important to blacklist these operators to avoid exporting
```

Comment:
@houseroad, this is the code for the blacklisting.
_black_list_in_opset() is defined in symbolic_helper.py.
These lines will redefine the ops in black_listed_operators  to raise a warning.

I am adding a test test_blacklisted_ops() in test_verify to test this, in https://github.com/pytorch/pytorch/pull/20036.

Reply:
https://github.com/onnx/onnx/blob/master/onnx/defs/operator_sets.h#L540

You can find the full list of changed ops in opset 10 here

Reply:
adding maxpool, averagepool and dropout.
The other ops are not in opset 9.